crimsobot/utils/image.py

```diff
@@ -258,6 +259,25 @@ async def pingbadge(ctx: Context, user_input: Optional[str], position: int) -> B
     return image_to_buffer(img, 'PNG')
 
 
+# async def lateralus_cover(ctx: Context, user_input: Optional[str]) -> BytesIO:
```

Comment:
just go ahead and delete this, if it's ever needed again it can be gotten from git history :)